docs/user/tutorial/40-people.rst

```diff
@@ -49,7 +49,11 @@ To create a new Person:
         Description
             Freda oversees all technical activities in our workshop.
 
-    .. todo:: screenshot of above
+.. image:: /user/tutorial/images/create_new_person.png
```

Comment:
I meant not by cropping important information, but by resizing the CKEditor and dialog box etc before making the screenshot - possible? Sometimes these elements have minimum heights so it's not always possible.